website/screens/components/flex/usage/FlexUsagePage.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  DxcFlex,
+  DxcParagraph,
+  DxcBulletedList,
+} from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+
+const sections = [
+  {
+    title: "Axes",
+    content: (
+      <>
+        <DxcParagraph>
+          Flexboxes are comprised mainly of two axes: the main axis and the
+          cross axis. The main axis is defined by the flex-direction property,
+          and the cross axis runs perpendicular to it.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            The main axis is defined by flex-direction, which has four possible
+            values: <Code>row</Code>, <Code>row-reverse</Code>,{" "}
+            <Code>column</Code> and <Code>column-reverse</Code>.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            The cross axis runs perpendicular to the main axis.
+            <DxcBulletedList type="circle">
+              <DxcBulletedList.Item>
+                If the flex-direction (main axis) is set to row or row-reverse
+                the cross axis runs down the columns.
+              </DxcBulletedList.Item>
+              <DxcBulletedList.Item>
+                If the flex-direction is set to column or column-reverse, the
+                cross axis runs down the rows.
+              </DxcBulletedList.Item>
+            </DxcBulletedList>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+      </>
+    ),
+  },
+  {
+    title: "Start and End",
+    content: (
+      <>
+        <DxcParagraph>
+          The flexbox makes no assumption about the writing mode of the
+          document. Left to right or right to left can be used depending, for
+          example, on the language used. A start and end edge is used to refer
+          to the direction of the placement of elements.
+        </DxcParagraph>
+        <DxcParagraph>
+          For example, If the flex-direction is row and the language is English,
+          then the start edge of the main axis will be on the left, the end edge
+          on the right.
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Flexbox Container",
+    content: (
+      <>
+        <DxcParagraph>
+          An area of a document laid out using flexbox is called a flex
+          container.
+        </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            Items display in a row (the flex-direction property&#39;s default is
```

Comment:
Maybe to put `The items` like in the following points?

Reply:
Done